build.gradle

```diff
@@ -1,23 +1,24 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 ext {
```

Comment:
a couple of comments for this section:
 - we don't need `minProviderSdkVersion` please remove it
 - `targetSdkVersion` should be 25
 - `zendeskSdkVersion` should be 1.7.3.1
 - `zopimSdkVersion` should be 1.3.2.1